tests/test_modeling_tf_common.py

```diff
@@ -446,6 +446,23 @@ def test_inputs_embeds(self):
 
             model(inputs_dict)
 
+    def test_resize_token_embeddings(self):
+        if not self.test_resize_embeddings:
+            return
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        INPUT_SHAPE = [1, 10, config.hidden_size]
+        for model_class in self.all_model_classes:
+            for size in [config.vocab_size - 10, config.vocab_size + 10, None]:
+                # build the embeddings
+                model = model_class(config=config)
+                emb_old = model.get_input_embeddings()
+                emb_old.build(INPUT_SHAPE)
+                # reshape the embeddings
+                new_embeddings = model._get_resized_embeddings(emb_old, size)
+                # # check that the the resized embeddings size matches the desired size.
+                assert_size = size if size is not None else config.vocab_size
+                self.assertEqual(new_embeddings.shape[0], assert_size)
+
```

Comment:
I think this test is interesting, but isn't testing everything. It would be great if you could do one similar too what is done with the [PyTorch implementation](https://github.com/huggingface/transformers/blob/master/tests/test_modeling_common.py#L520), like verifying that the weights are identical after resizing. Would it be possible for you to do so? After this, we'll be ready to merge!

Reply:
Yes, I had this in the original BERT test. I will introduce it here.

Reply:
Great!

Reply:
please take a look!